tests/dummy/app/index.html

```diff
@@ -14,6 +14,16 @@
     <link rel="stylesheet" href="assets/vendor.css">
     <link rel="stylesheet" href="assets/dummy.css">
 
+    <script type="text/javascript"
+        src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.6.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
+    </script>
+    <script type="text/x-mathjax-config">
```

Comment:
Sometimes we'll use a CDN in prod but a packaged dependency locally. (to make debugging easier) Eg:
https://github.com/CenterForOpenScience/ember-preprints/blob/develop/ember-cli-build.js#L62

The biggest change in my thinking since the last review has been our interest in automated tools for dependency management. Things like Snyk (security scanner) and Greenkeeper (package.json updater) rely on these explicit file listings of dependencies to figure out what your project uses, and that makes hard-coded URLs less attractive. 

We're increasingly noticing the challenge of finding and incorporating updates as we do this more. To give you a concrete example? The [latest version](http://docs.mathjax.org/en/latest/whats-new-2.7.html) of MathJax is actually 2.7. (Oct 2016)